### DISCUSSION TITLE
contributon

### DISCUSSION THREAD

**Discussion by Satyam1923** [zed-industries/zed#8380] [Feature Requests]
I want to start my open source career but i dont know from which field to start ? 
Can anyone help

**Comment by Moshyfawn** [comment]
I describe where to start looking for problems to solve in Zed in https://github.com/zed-industries/zed/discussions/8309#discussioncomment-8573305.

As for the whole "open source career" idea - I wouldn't advise thinking of it as a career. It's first and foremost a hobby, a way to learn and help others.

You can start by using software like Zed. It's important to understand the area you want to contribute to. Then you can start by fixing small problems you've encountered or adding small features you've missed; it's much more productive to start with something you're familiar with, rather than just trying to contribute to some random project.
